man/systemd-sysusers.xml

```diff
@@ -94,6 +97,22 @@
         paths. </para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--target=<replaceable>PATH</replaceable></option></term>
```

Comment:
Hmm. It is not easy for me to understand the logic from the name '--target='. How about '--priority-of=' or something, as only priority of the .conf file search logic is used. The name '--target=' is too abstract or ambiguous for me.

Reply:
The name comes from "this configuration will land (is targeted to) file ...". I don't like "priority-of", what about `--destination` or `--replaces`?

Reply:
For me, `--replaces=` seems fine.